htdocs/PI/write/PIWriteRequest.php

```diff
@@ -74,6 +74,10 @@ class PIWriteRequest {
     #An array that will ultimately be returned to the user
     private $returnObject=null;
 
+    #An array for generic exception messages
```

Comment:
This comment is pretty much the same as the line below it, could you perhaps explain *why* you are creating an array for generic exception messages?

Reply:
Agreed, resolved in latest version